common/transport/mqtt/src/main/java/org/thingsboard/server/transport/mqtt/MqttTransportHandler.java

```diff
@@ -1330,6 +1341,40 @@ public void onToDeviceRpcRequest(UUID sessionId, TransportProtos.ToDeviceRpcRequ
         }
     }
 
+    private void onGetServiceConfigurationRpc( TransportProtos.SessionInfoProto sessionInfo, ChannelHandlerContext ctx, int msgId, TransportProtos.ToServerRpcRequestMsg rpcRequestMsg) {
+        TenantId tenantId = TenantId.fromUUID(new UUID(sessionInfo.getTenantIdMSB(), sessionInfo.getTenantIdLSB()));
```

Comment:
Let's just create a field of tenantId in the DeviceAwareSessionContext, next to deviceId. (Assign it in the setDeviceInfo)